cmd/common.go

```diff
@@ -91,7 +91,7 @@ func runChecks(nodetype check.NodeType, testYamlFile string) {
 
 	// Checks that the executables we need for the section are running.
 	if err != nil {
-		exitWithError(fmt.Errorf("failed to get a set of executables needed for tests: %v", err))
+		glog.V(1).Info(fmt.Sprintf("failed to get a set of executables needed for tests: %v", err))
```

Comment:
Can we still just continue with running the tests in this section? Or should we be returning from this function with this error? I guess we should be returning [FAIL] or [WARN] for each test in this section

Reply:
Its before running anything, its changing the behavior from not running and run and fail/warn on the relate tests 
This is the output before the change: 
![image](https://user-images.githubusercontent.com/12915002/93433720-d85b4400-f8cf-11ea-9975-67cc45b8e8d8.png)
After the change we can see that now we can run and get cis output results:
![image](https://user-images.githubusercontent.com/12915002/93433881-12c4e100-f8d0-11ea-9e9a-4d501f4cc5fd.png)